backend/api/requests.py

```diff
@@ -205,35 +205,41 @@ async def delete_request(
 
 
 @router.post("/invite", status_code=status.HTTP_201_CREATED)
-async def invite_user_to_group(
-        request: api_request_classes.InviteToGroupRequest,
+async def invite_users_to_group(
```

Comment:
why don't you use the single one many times?

Reply:
because it is more efficient to do a single query with insert_many